content/content-strategy/audit-content/action-audit/index.yml

```diff
@@ -0,0 +1,24 @@
+pagetitle: Action your audit
+description: Abstract
+weight: 80
+layout: layout/topic
+theme: blue
+header:
+  - /_shared/globalheader.md
+  - /_shared/header.md
+main:
+  - intro.md
+  - toc.md #toc
+  - heading.md #report findings
+  - section.md #report findings
+  - example.md #example
+  - section.md #report findings2
+  - example2.md #example
+  - heading2.md #remove content
+  - section2.md #remove content
+  - heading3.md #archive
+  - section3.md #archive
+  - heading4.md #rewrite
+  - section4.md #rewrite
```

Comment:
Looks like a work in progress. Please add `- /_shared/feedback.md` to the end of the list in `main`

Reply:
Trisha fixed